### PR TITLE
Destroy api

### PR DESCRIPTION
**Pull request by fde31**
As mentioned in #31 the API misses a destroy function to gracefully remove lory from a DOM element. I added this, a simple Karma test and some documentation about it with this PR. In order to check if I got rid of every reference/listener I used the following simple HTML file ([see Gist](https://gist.github.com/fde31/59ccb1e31206d040418c)) and Chrome's Dev tools. The screenshots of the profile and and timeline output are included in the Gist.

### CODE REVIEW

File: test/lory.test.js [hunk 1]
```diff
@@ -169,3 +169,23 @@ describe('.prev()', function () {
         assert.equal(instance.returnIndex(), 0);
     });
 });
+
+describe('.destory()', function () {
```
Comment (Cordazar) [inline test/lory.test.js]: spelling of destroy()

File: README.md [hunk 2]
```diff
@@ -251,6 +255,10 @@ grunt karma
         <th colspan="2">on.lory.resize</th>
         <td>fires on every resize event</td>
     </tr>
+    <tr>
+        <th colspan="2">on.lory.destory</th>
```
Comment (Cordazar) [inline README.md]: spelling of destroy

**Comment by fde31** [comment]
ouch. Sorry for that oversight weird typos should be fixed now.

**Comment by Cordazar** [comment]
:+1: Looks very good to me :)

**Pull request merged**